tests/test_get_avatar.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import json
+import requests as r
+import yaml
+import pytest
+from parameterized import parameterized
+import logging
+import username_api
+import os.path
+
+data = yaml.load(open(os.path.join('tests', 'test_data.yml')))
+websites = yaml.load(open('websites.yml'))
+
+
+def load_test_cases(type):
+	res = []
+	for website in data:
+		if type == 'with':
+			if 'avatar_usernames' in data[website]:
+				users = data[website]['avatar_usernames']
+			else:
+				users = data[website]['taken_usernames']
+		else:
+			users = data[website]['available_usernames']
+		res.extend((website, user) for user in users)
+
+	return res
+
+
+def custom_name_func(testcase_func, param_num, param):
+	return '%s_%s' % (
+		testcase_func.__name__,
+		parameterized.to_safe_name('_'.join(str(x) for x in param.args)),
+	)
+
+
+class TestGet_avatar(object):
+
+	@parameterized.expand(load_test_cases('with'),
+						  testcase_func_name=custom_name_func)
+	def test_with_avatar(self, website, user):
+		if website == 'behance':
```

Comment:
this belongs in `load_test_cases`

Reply:
So we shouldn't even load those test cases? I'd rather keep a "skipped" notification no?

Reply:
you can generate a skipped test in `load_test_cases`